### PR TITLE
[Feature #156248366]Add methods on the books model

### PR DESCRIPTION
**Pull request by Krafty-Coder**
### Done
On the book models I created 

- add
- modify
- delete 
- get by id
- get all

story link https://www.pivotaltracker.com/story/show/156248366

### CODE REVIEW

**Comment by Krafty-Coder** [comment]
Add methods on book model

File: app/book_model.py [hunk 1]
```diff
@@ -1,15 +1,72 @@
...
+        self.number_of_books = number_of_books
+
+    def delete_book(self, id):
+        for book in library:
+            if book[id] == id:
+                library.remove(book)
+
+        return "book removed successfully, {} books remaining".format(len(library))
+
+    def add_book(self, title, author, id, publisher, number_of_books):
```
Comment (rosemaina) [inline app/book_model.py:20]: This way you are adding the book is not following the OOP standards. You are supposed to create an instance of the class when creating a new book.

**Comment by rosemaina** [comment]
@Krafty-Coder

**Review by rosemaina — COMMENTED** [review]
Have a central place for saving your data structure say a `data.py` then have `user.register_user()` save the data there.

**Comment by Krafty-Coder** [comment]
@rosemaina 
Noted, I will make the changes

**Pull request merged**